packages/hooks/useMaxResolution.ts

```diff
@@ -40,7 +40,8 @@ export const useMaxResolution = ({
   }, [windowWidth, isSidebarExpanded, noMargin, responsive, isMobile]);
 
   return {
-    width: width > screenContentMaxWidth ? screenContentMaxWidth : width,
+    width:
+      width > screenContentMaxWidthLarge ? screenContentMaxWidthLarge : width,
```

Comment:
Please, let `screenContentMaxWidth` here and add a property `isLarge` or something to `useMaxResolution` if you want to use `screenContentMaxWidthLarge`.
We need 1092px as max width for many screens to respect the design.
This hook starts to look weird, but we need to be careful with it for now.
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/01be9d81-5be0-49e7-adfa-9ba1c627e339)